sdk/aqueduct/integrations/integration.py

```diff
@@ -25,6 +25,20 @@ def describe(self) -> None:
         }
         print(json.dumps(description_map, sort_keys=False, indent=4))
 
+    def is_relational(self) -> bool:
+        """Returns whether the integration connects to a relational data store."""
+        return self.service in [
+            ServiceType.POSTGRES,
+            ServiceType.SNOWFLAKE,
+            ServiceType.MYSQL,
+            ServiceType.REDSHIFT,
+            ServiceType.MARIADB,
+            ServiceType.SQLSERVER,
```

Comment:
Reminder to add service type ATHENA. Although since this helper is currently only relevant to load operators, Athena shouldn't be relevant.